pkg/controller/operatingsystemconfig/os-common/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
```

Comment:
remove whole chart - no actual controller developed here.

Reply:
Are templates. It is explained in the README.

Reply:
Nobody will maintain this. It will run out of date. Let's not have this Helm chart here. If somebody wants to develop a new OS controller he/she can easily copy the Helm chart of another existing controller which is actually working and up-to-date.

Reply:
Removed